app/forms/project_form.rb

```diff
@@ -66,4 +66,15 @@ def invite_to_github_team
     username = user.nickname
     github.invite_to_team(team_id, username)
   end
+
+  def create_and_fork_repo
```

Comment:
Assignment Branch Condition size for create_and_fork_repo is too high. [16.12/15]